engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -1359,7 +1409,8 @@ private void addDAGToCurrentDAG(ModuleMeta moduleMeta)
     String name;
     for (OperatorMeta operatorMeta : subDag.getAllOperators()) {
       name = subDAGName + MODULE_NAMESPACE_SEPARATOR + operatorMeta.getName();
-      this.addOperator(name, operatorMeta.getOperator());
+      Operator op = this.addOperator(name, operatorMeta.getOperator());
```

Comment:
Can LogicalPlan class provide a new method that allows to add Operator with existing OperatorMeta? Something like 
```
  private <T extends Operator> T addOperator(final String name, final T operator, final OperatorMeta meta)
  {
    
  }
```